packages/web-components/fast-foundation/src/button/button.template.ts

```diff
@@ -6,7 +6,7 @@ import { Button } from "./button";
  * @public
  */
 export const ButtonTemplate = html<Button>`
-    <template class="${x => x.appearance}">
+    <template>
```

Comment:
You don't need the `<template>` element if you're not adding any attributes to it.